src/Ledger/Deleg.lagda

```diff
@@ -239,6 +239,11 @@ instance
   Computational-POOL = fromComputational' Computational'-POOL
 
 instance
+  Dec-⊎ : ∀ {a b} {A : Set a} {B : Set b} → ⦃ Dec A ⦄ → ⦃ Dec B ⦄ → Dec (A ⊎ B)
+  Dec-⊎ ⦃ yes p ⦄ ⦃ _     ⦄ = yes (inj₁ p)
+  Dec-⊎ ⦃ no _  ⦄ ⦃ yes q ⦄ = yes (inj₂ q)
+  Dec-⊎ ⦃ no ¬p ⦄ ⦃ no ¬q ⦄ = no λ { (inj₁ p) → ¬p p; (inj₂ q) → ¬q q }
+
```

Comment:
Had to move `Dec-⊎` here, where it is used, so that it doesn't conflict with one of the `_≤_` instances that we want Agda to use in the `PPUp.Properties` module.